pkg/redis/redis_test.go

```diff
@@ -577,7 +579,7 @@ func TestMutex(t *testing.T) {
 	}
 	a.So(lockTTL, should.BeGreaterThan, 0)
 	a.So(lockTTL, should.BeLessThanOrEqualTo, ttl)
-	a.So(listTTL, should.BeLessThanOrEqualTo, lockTTL)
+	a.So(listTTL, should.BeLessThanOrEqualTo, ttl)
```

Comment:
The `listTTL` is actually reset to `ttl` on mutex unlock, not the TTL of the lock itself. This is according to the implementation.